scripts/linux/runE2ETest.sh

```diff
@@ -714,7 +719,7 @@ function run_longhaul_test() {
         --initialize-with-agent-artifact "$INITIALIZE_WITH_AGENT_ARTIFACT" \
         --leave-running=All \
         -l "$deployment_working_file" \
-        --runtime-log-level "Info" \
+        --runtime-log-level "$RUNTIME_LOG_LEVEL" \
```

Comment:
I think you don't need to set runtime log level here and deployment json, only 1 place is enough.  And I think it is better to change here instead of deployment file; as it doesn't require to update deployment json files.